lib/mix/tasks/docs.ex

```diff
@@ -200,6 +210,68 @@ defmodule Mix.Tasks.Docs do
       where path is either an relative path from the cwd, or an absolute path. The function
       must return the full URI as it should be placed in the documentation.
 
+  ## Encrypted debug info
+
+  If a module is compiled with [encrypted debug info](`:compile.file/2`), ExDoc will not be able to
+  extract its documentation without first setting a decryption function or utilizing
+  `.erlang.crypt` as prescribed by `m::beam_lib#module-encrypted-debug-information`. Two
+  convenience options (see below) are provided to avoid having to call `:beam_lib.crypto_key_fun/1`
+  out-of-band and/or to avoid using `.erlang.crypt`.
```

Comment:
This looks great, but let's support only the `.erlang.crypt` style for now. The reason for this is because erlang.config files cannot define functions (afaik), so it is not a feature they would be able to use anyway. Let's start with the minimum necessary and expand on that later. WDYT?

Reply:
To clarify, you want documentation about `.erlang.crypt`, but all the other changes should be reverted? Sounds good to me. 

Do you want me to add tests or is that not necessary? Adding a test for this could be invasive - either putting a `.erlang.crypt` file in the root project directory or messing with the `XDG_CONFIG_HOME` environment variable. I'm leaning toward no, what do you think?